#!/usr/bin/env python
# -*- coding: utf-8 -*-
# @Time    : 11/5/16 6:56 PM
# @Author  : YangGao
# @Site    : 
# @File    : __init__.py.py


# list_str = '[5,3.7,9,12]'
# print(list_str)
# list_str = eval(list_str)
#
# print(list_str)
#
# print(list_str[2])
# '''
# code: 4
# code 5<4
# False
#
# '''
# x = input('code: ')
# '''just for number'''
# check_out = eval(input('code'))
# print(check_out)

# exec('print("so this works like eval")')
#
# list_str = exec(list_str)
# '''
#    so this works like eval
#    None
# '''
# print(list_str)
#
# exec('list_str2 =[3,4.5,3,8]')
# '''
# [3, 4.5, 3, 8]
# '''
# # don't worry about the error mentioned ,
# # codes will be running well
# print(list_str2)
